src/components/shared/protected/protectedError.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+
+import { ErrorMsg } from '../forms';
+import { joinClasses } from 'utilities';
+
+import './protectedError.css';
+
+export const ProtectedError = ({ permission, t, className }) => (
+  <div className={joinClasses('protected-error-container', className)}>
```

Comment:
I'm not convinced you need the extra div around `ErrorMsg`. You can already pass extra class names to ErrorMsg itself to customize it. Seems like that is likely the way to go.

Reply:
Agreed. However, I followed the convention from AjaxError... which does include the extra div.